frontend/src/container/NewDashboard/ComponentsSlider/menuItems.ts

```diff
@@ -18,8 +18,11 @@ export type ITEMS = 'TIME_SERIES' | 'VALUE';
 
 interface ItemsProps {
 	name: ITEMS;
-	Icon: () => JSX.Element;
+	Icon: (props: colorProps) => JSX.Element;
 	display: string;
 }
 
+interface colorProps{
```

Comment:
feel like this is repetitive 

can we import from icon and use here ?

Reply:
ok